catalog/app/components/Preview/loaders/Tabular.tsx

```diff
@@ -0,0 +1,191 @@
+import * as R from 'ramda'
+import * as React from 'react'
+
+import { HTTPError } from 'utils/APIConnector'
+import * as AWS from 'utils/AWS'
+import * as Config from 'utils/Config'
+import * as Data from 'utils/Data'
+import { mkSearch } from 'utils/NamedRoutes'
+import type { S3HandleBase } from 'utils/s3paths'
+
+import { CONTEXT, PreviewData } from '../types'
+
+import * as Csv from './Csv'
+import * as Excel from './Excel'
+import * as Parquet from './Parquet'
+import * as utils from './utils'
+
+const isJsonl = R.pipe(utils.stripCompression, utils.extIs('.jsonl'))
+
+export const detect = R.anyPass([Csv.detect, Excel.detect, Parquet.detect, isJsonl])
+
+// TODO: ['bed', 'fcs', 'ipynb', 'vcf']
```

Comment:
why are these in TODO? they don't seem like a general-purpose tabular data

Reply:
I copied them from API stub, but you are right, I'll remove them